mithril-stm/src/single_signature/mod.rs

```diff
@@ -0,0 +1,5 @@
+mod signature;
+mod signature_registered_party;
+
+pub use crate::single_signature::signature::StmSig;
+pub use crate::single_signature::signature_registered_party::StmSigRegParty;
```

Comment:
The filter on what is public is done in the `lib.rs` file, so this will be more readable:
```suggestion
pub use signature::*;
pub use signature_registered_party::*;
```